testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/configs/ConfigHdp3.java

```diff
@@ -56,6 +57,7 @@ public void extendEnvironment(Environment.Builder builder)
                         "/docker/presto-init.d/presto-init-hdp3.sh");
             }
         });
+        builder.configureContainer(HADOOP, container -> container.withEnv("TZ", "/usr/share/zoneinfo/UTC"));
```

Comment:
what is this for?

Reply:
When doing implicit coercion to timestamp, hive cast to a zone specific String - so wanted it to be in sync with UTC (like we do in Trino). This is only specific to ORC file for other formats it doesn't consider the timezone of the HiveServer.

Reply:
quick google search suggests that `TZ` env variable means time zone name. `/usr/share/zoneinfo/UTC` looks like a file path, not a time zone name. How does it work?

> When doing implicit coercion to timestamp, hive cast to a zone specific String - so wanted it to be in sync with UTC (like we do in Trino).

would this information be useful for future code maintainers?

Reply:
Quick google search suggested me to set `TZ` to this file and it worked for me. Is there any workaround I could try here.

Reply:
> Quick google search suggested me to set TZ to this file


interesting. i should have taken a note what was the search terms in my quick google search...

what happens if you
```
container.withEnv("TZ", "UTC")
```

(instead of the file)?

and what happens if you
```
container.withEnv("TZ", "/dev/null")
```

or
```
container.withEnv("TZ", "absolutely-random-string")
```
?

Reply:
It looks like specifying a timezone also works like passing the file
```
container.withEnv("TZ", "UTC")
```

If we pass 
```
container.withEnv("TZ", "/dev/null")
```

It switches to default value  `UTC` 

For 
```
container.withEnv("TZ", "absolutely-random-string")
```

It switches to default value `UTC`.

Reply:
> If we pass
> 
> ```
> container.withEnv("TZ", "/dev/null")
> ```
> 
> It switches to default value `UTC`
> 
> For
> 
> ```
> container.withEnv("TZ", "absolutely-random-string")
> ```
> 
> It switches to default value `UTC`.

so `/usr/share/zoneinfo/UTC` looks like something meaningful, but actually is "garbage time zone" and causes something (the JVM?) to switch to UTC.

funny